src/pages/Home/Home.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import getToken from '../../lib/tokens';
+
+const saveToken = () => {
+  const token = getToken();
+  localStorage.setItem('token', token);
+  window.location.reload(false);
```

Comment:
FYI I got the following error that `window.location.reload` has been deprecated when I ran this locally

![image](https://user-images.githubusercontent.com/61371242/126730613-d343a555-ca62-4478-8e28-f6b0da1be27a.png)

Reply:
Just saw that Ander also commented on this. I'll close this comment!